packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -6,6 +6,8 @@ import {
 import { IResizeGroupProps } from './ResizeGroup.Props';
 import styles = require('./ResizeGroup.scss');
 
+const RESIZE_DELAY = 16;
```

Comment:
Any particular reason for using `16ms` ? If resize is user initiated we can keep something ~ `100ms` which will have throttling effect with no major noticeable re-render delay.